.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  CI: true
+
+jobs:
+  test:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [10, 12, 14]
```

Comment:
Thoughts on picking one version of Node? I realize their LTS factors into this, and v10 is falling off soon, but my goal is to keep this as light as can be.